packages/core/src/lib/s3/s3-repository.ts

```diff
@@ -0,0 +1,29 @@
+import { type PresignedPost, createPresignedPost } from "@aws-sdk/s3-presigned-post"
+import { S3Client } from "@aws-sdk/client-s3"
+import { env } from "@dotkomonline/env"
+
+export interface S3Repository {
+  getPresignedPostData(bucket: string, filename: string, mimeType: string, maxSizeMB: number): Promise<PresignedPost>
```

Comment:
what about renaming this to createPresignedUploadUrl?

Reply:
Sure!